spyder/config/utils.py

```diff
@@ -119,7 +119,7 @@ def get_edit_filetypes():
     other_exts = ['.ipynb', '.md']
     all_exts = tuple(pygments_exts + other_exts)
     text_filetypes = (_("Supported text files"), all_exts)
-    return [text_filetypes] + EDIT_FILETYPES
+    return EDIT_FILETYPES + [text_filetypes]
```

Comment:
Hi Gonzalo, I decided to switch line (#122) so Spyder will suggest in comboBox to save the file with the extensions .py, .pyw, .ipv instead *.6pl, *.6pm, *:ABAP

I consider the most programmers that use Spyder need this suggestion by default instead search it at last of ComboBox  extension list.